### DISCUSSION TITLE
Geräteübersicht ohne meshd & nexus?

### DISCUSSION THREAD

**Discussion by marvinlehmann** [Freetz-NG/freetz-ng#897] [Allgemein]
Moooin,

nachdem ich immer mehr AVM-Kram von meiner 7590 geworfen habe, ist mir aufgefallen,
dass die Auflistung unter _Heimnetz -> Mesh/Netzwerk_ und damit auch die Liste der Heimnetzgeräte auf der Startseite nicht mehr geht. Verantwortlich sind wohl meshd (schon naheliegend..) für WLAN-Geräte und meshd+nexus(??!) für LAN-Geräte.
Sind die Dienste noch installiert und man stoppt sie im Betrieb, werden in diesem Fall die Listen einfach nicht mehr aktualisiert.
Nach durchforsten der wunderschönen Lua-Skripte scheint `"landevice settings/landevice/list(active,...)"` für die Listen ausgelesen zu werden, wobei _active_ unverändert bleibt. TR064 setzt ja ebenfalls auf diese Queries.

Geräte im WLAN kann ich ja weiterhin unter _WLAN -> Funknetz_ einsehen aber LAN Geräte (+Ports) wohl nirgends?
Ich schätze Mal, dass man da nix machen kann und ich mit dem "Mesh" leben muss?
Oder kann ich mir diese Daten noch von woanders holen (außer arp & ip n) und mir vielleicht ne eigene Übersicht basteln bzw. versuchen das Webinterface zu patchen?

Und auch mal vielen Dank für dieses tolle Projekt! :)

**Comment by fda77** [comment]
Grob: "nexus" war der daemon in alten FOS versionen als damit nur die AHA Geräte im Lan verteilt wurden, da hat jedes avm Gerät alle Sensoren+Aktoren im Lan gesehen, sowas gibt es heute nicht mehr. Später kam dann das "mesh" mit dem Telefonie-+Wlan-Daten im Lan verteilen dazu. Ich hab das ganze meist aus der Firmware geworfen und kenne mich nicht weiter damit aus. Vor diesem ganzen mes/nexus gab es die Lan-Geräte aber auch, da hatte der multid dies vermutlich gesammelt - falls es diesen Code noch gibt könnte man ihn evtl irgend wo wieder aktivieren...
Jedenfalls fehlern bei mit auch alle Netzwerkkgeräte, was mit bislang aber nicht aufgefallen ist. Zum einen weil dort auch Dect und USB Geräte aufgelistet werden, oder da die Fritzbox reinem DSL-Modem am Wan-Port von richtigen Routern ist und es keine anderen Geräte gibt.

Wenn ich diese Übersicht unbedingt brauchen würde, wäre es eine gute Idee die Ausgaben von arp oder  ip via cron zb alle 5 Minuten zu parsen und damit manuell avm zu updaten
https://freetz-ng.github.io/freetz-ng/wiki/60_Development/uimods.html#listen kennst du?
```
ctlmgr_ctl u landevice
...
```
und
```
ctlmgr_ctl l landevice "settings/landevice/list(active,ip)"
        landevice0      0       192.168. ...
        landevice1      0       192.168. ...
        landevice2      0       192.168. ...
        landevice3      1       192.168. ...
        ...
```

**Reply by marvinlehmann** [reply]
Vielen Dank für die Erklärung!

Die Seite kannte ich noch nicht aber über den ctlmgr_ctl bin ich schon gestolpert.
Leider scheint das Attribut schreibgeschützt zu sein. (`ctlmgr_ctl w landevice settings/landevice3/online 1` bringt nix)
Auch lässt sich  _/ethernetport_ nicht setzen, falls man diesen anderweitig herausfinden würde.

**Comment by fda77** [comment]
Ich habe auch etwas experimentiert, "active" und "online" konnte ich auch nicht ändern, diese scheinen nur zur Anzeige zu sein. Evtl müssen auch mehrere Variablen zusammen gesetzt werden, zB beim anlegen eines neuen Benutzer reicht es nicht nur den Namen anzugeben, sondern ein Passwort muss auch gesetzt werden. Ich glaube unsichere Passwörter wurden sogar abgelehnt.
Vielleicht ist es einfacher in die .lua gleich "arp -a " auszuführen und das Ergebnis entsprechend zu parsen, falls du lua kannst
